application/classes/Ushahidi/Formatter/Post/GeoJSONCollection.php

```diff
@@ -47,6 +47,20 @@ public function __invoke($entities)
 
 			if (! empty($geometries))
 			{
+				$set_name = [];
+				$set_ids = $entity->sets;
+
+				if ($set_ids != null) {
+					$query = DB::select('sets.name')
```

Comment:
That'd be great. If you can also include URL that'd be perfect
We use ```URL::site(Ushahidi_Rest::url($resource, $id), Request::current())``` elsewhere to generate URLs.